pkg/pkgrepository/package_repo_app.go

```diff
@@ -100,7 +102,9 @@ metadata:
     kapp.k14s.io/disable-original: ""
     #@overlay/match missing_ok=True
     kapp.k14s.io/disable-wait: ""
-`,
+    #@overlay/match missing_ok=True
+    packaging.carvel.dev/pkgr: %s
```

Comment:
lets name it `packaging.carvel.dev/package-repository-ref` and value of it would be `repo-namespace/repo-name`.